analysis/render.py

```diff
@@ -248,6 +248,12 @@ def fmt_control_value(row: pd.Series):
     )
 
 
+def strip_commit_hash(description: str) -> str:
+    """Strip commit hash from a metric description"""
+    marker = "Commit hash:"
+    return description.split(marker)[0].strip()
```

Comment:
Hard dependency on the commit hash being the suffix of the description/anything after the commit hash also being internal content that should be hidden. This matches https://github.com/Azure/online-experimentation-deploy-metrics/pull/13/files current behavior, and risk of getting out of sync is acceptably low.